src/NuGet.Clients/NuGet.VisualStudio.Client/source.extension.vs14.vsixmanifest

```diff
@@ -23,6 +23,7 @@
     <Asset Type="Microsoft.VisualStudio.MefComponent" d:Source="Project" d:ProjectName="NuGet.Console" Path="|NuGet.Console|" />
     <Asset Type="Microsoft.VisualStudio.MefComponent" d:Source="Project" d:ProjectName="NuGet.VisualStudio.Implementation" Path="|NuGet.VisualStudio.Implementation|" />
     <Asset Type="Microsoft.VisualStudio.MefComponent" d:Source="Project" d:ProjectName="NuGet.SolutionRestoreManager" Path="|NuGet.SolutionRestoreManager|" />
+    <Asset Type="Microsoft.VisualStudio.MefComponent" d:Source="Project" d:ProjectName="NuGet.VisualStudio.Common" Path="|NuGet.VisualStudio.Common|" />    
```

Comment:
why add here? Should we continue to manage dev14?

Reply:
larger question. Unless we decide to remove this, I think this is okay.

Reply:
We should remove everything dev 14 related from the codebase.

https://github.com/NuGet/Home/issues/7112